elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -242,12 +232,12 @@ $variables: (
 
 // Default link background focus color
 :host(:not([pfe-priority]):not([aria-disabled="true"])) {
-  --pfe-cta--Color: #{pfe-apply-broadcast(link)};
+  --pfe-cta--Color: #{pfe-broadcasted(link)};
```

Comment:
One more big picture question that doesn't have to be solved in this branch: 
because of the way we had to style the default CTA, the only way to use `  --pfe-cta--Color` is to attach it to a `pfe-cta` element. You can't put it on the body or html tag because this always wins:

![image](https://user-images.githubusercontent.com/456863/83267691-97a9b480-a192-11ea-957d-cd2e3cbaf046.png)

Really this is a good use case of why we need this whole PR, because this branch is solving for this exact problem everywhere but pfe-cta. Just wanted to flag that its still an issue here. Again, not sure what we can do about it. I'm just surfacing the thoughts bubbling up around this just to make sure we're on the same page & I'm not missing the boat. :)